### DISCUSSION TITLE
How to add "chips"?

### DISCUSSION THREAD

**Discussion by hoechenberger** [squidfunk/mkdocs-material#4725] [Q&A] [answered]
Hello,

I am looking for a way to add what is called "chips" in Material:
https://m3.material.io/components/chips/overview

I got inspired by the Seaborn [changelog](https://seaborn.pydata.org/whatsnew/v0.12.1.html
), where each change is "tagged" with one or multiple such chips:

<img width="450" alt="Screenshot 2022-12-10 at 15 40 28" src="https://user-images.githubusercontent.com/2046265/206860801-0e1b590b-5325-46d0-af05-28535ce392c4.png">

Would there be a way to achieve something similar with Material for MkDocs?

Thank you very much!

Richard

**Comment by facelessuser** [comment, marked as answer]
I could see this easily being done with existing plugins and a little CSS. Using the `attr_list` extension you could add a special class to something like a bold word:`**Feature**{.chip-feature}`. Then you just need a little styling.

**Reply by squidfunk** [reply]
For starters, you could use the following [additional CSS](https://squidfunk.github.io/mkdocs-material/customization/#additional-css):

``` css
.chip-feature {
  display: inline-block;
  background: deeppink;
  color: white;
  padding: 0px 16px;
  border-radius: 20px;
}
```

<img width="110" alt="Bildschirmfoto 2022-12-10 um 18 58 11" src="https://user-images.githubusercontent.com/932156/206868857-4784fae6-d42d-4c89-8d05-267644acfd12.png">

**Comment by hoechenberger** [comment]
Thank you, this actually solves my problem!
